scripts/utilities/update_revision.sh

```diff
@@ -158,8 +158,6 @@ get_latest_revision ()
 		# relocated within the vendors directory.
 		if [[ $package == "gcc" ]] && [[ $configset != "next" ]]; then
 		    branch="refs/vendors/ibm/heads/${branch}"
-		else
```

Comment:
Ok. This `else` should not be here because it is overwriting the value we have set above when `ATSRC_PACKAGE_BRANCH` is not set.